packages/axios-extension/src/abap/adt-catalog/services/ui5-rt-version-service.ts

```diff
@@ -0,0 +1,48 @@
+import { isAxiosError } from 'axios';
+import type { AdtCategory } from '../../types';
+import { AdtService } from './adt-service';
+import type { Logger } from '@sap-ux/logger';
+
+/**
+ * UI5RtVersionService implements ADT requests to get UI5 Version
+ * of ABAP system.
+ */
+export class UI5RtVersionService extends AdtService {
+    /**
+     * @see AdtService.getAdtCatagory()
+     */
+    private static adtCategory = {
+        scheme: 'http://www.sap.com/adt/categories/filestore',
+        term: 'ui5-rt-version'
+    };
+
+    // Instantiated by calling ServiceProvider.createService()
```

Comment:
Not required, will inherit from `AdtService`.

Reply:
Fixed

Reply:
Closing for @longieirl since he is ooo